### PR TITLE
Multitargeting NET6 and NET7

### PR DESCRIPTION
**Pull request by stsrki**
https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-7-rc-1/

Tested and working:

- demo app(s)
- docs app
- NuGet publish
- demo apps publish, wasm and server

---

The only thing that is not working out of the box is the Gutter, a value tuple parameter. The problem is in VS razor engine, so it is out of our reach to fix it. I have introduced extra parameters as a workaround.

### CODE REVIEW

**Comment by David-Moreira** [comment]
Looks good to me. 

Shame on the Gutter I guess.

I hadn't downloaded .NET7 yet.
Downloaded & compiled/ran docs & demo successfully.

File: Build/Blazorise.props [hunk 1]
```diff
@@ -25,9 +25,25 @@
...
+
+	<ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+		<PackageReference Include="Microsoft.AspNetCore.Components" Version="6.0.0" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.0" />		
+	</ItemGroup>
+
+	<ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
+		<PackageReference Include="Microsoft.AspNetCore.Components" Version="7.0.0-preview.7.22376.6" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.0-preview.7.22376.6" />
+	</ItemGroup>
```
Comment (David-Moreira) [inline Build/Blazorise.props:47]: Should this target the RC now?

Comment (David-Moreira) [reply Build/Blazorise.props:47]: nvm....

**Comment by David-Moreira** [comment]
Would go ahead and merge this in, and make available a new blazorise rc asap with this.

**Comment by stsrki** [comment]
> Would go ahead and merge this in, and make available a new blazorise rc asap with this.

Should we also include disposable thingy in the RC?

**Comment by David-Moreira** [comment]
I guess the sooner it's tested by the general community, the better. I am still going to test it with a blank app like we've talked.

**Pull request merged**